### ISSUE TITLE
Add ability to have multiple download files

### ISSUE THREAD

**Issue by mindctrl** [wpninjas/edd-git-download-updater#20]
Customer requested ability to have multiple download files like the standard EDD delivery method supports.

**Comment by kstover** [comment]
This is something that we would love to build in, but I'm not sure when we'll be able to add it. Our main project is taking up all of our time at the moment. :(

**Comment by mindctrl** [comment]
@kstover understandable. I notice the JS that hides the extra file row stuff is also hiding it on the upgrade paths box in Software Licensing. Was this intentional?

**Comment by kstover** [comment]
@mindctrl Yup. Since we only support one file at a time, there isn't a need to select anything there. We were afraid that it would just cause confusion.
